### PR TITLE
MODINV-326 - Refactor data-import handler to consume message from Kafka

### PR DESCRIPTION
**Pull request by RuslanLavrov**

### CODE REVIEW

File: .rancher-pipeline.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
+stages:
```
Comment (bohdan-suprun) [inline .rancher-pipeline.yml:1]: Please make sure this removed before merge to master.

Comment (RuslanLavrov) [reply .rancher-pipeline.yml:1]: yes sure

File: src/main/java/org/folio/inventory/DataImportConsumerVerticle.java [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+    DI_SRS_MARC_BIB_RECORD_MODIFIED, DI_SRS_MARC_BIB_RECORD_MODIFIED_READY_FOR_POST_PROCESSING,
+    DI_SRS_MARC_BIB_RECORD_MATCHED, DI_SRS_MARC_BIB_RECORD_NOT_MATCHED,
+    DI_INVENTORY_INSTANCE_CREATED, DI_INVENTORY_INSTANCE_UPDATED,
+    DI_INVENTORY_INSTANCE_MATCHED, DI_INVENTORY_INSTANCE_NOT_MATCHED,
+    DI_INVENTORY_HOLDING_CREATED, DI_INVENTORY_HOLDING_UPDATED,
+    DI_INVENTORY_HOLDING_MATCHED, DI_INVENTORY_HOLDING_NOT_MATCHED,
+    DI_INVENTORY_ITEM_CREATED, DI_INVENTORY_ITEM_MATCHED,
+    DI_INVENTORY_ITEM_NOT_MATCHED, DI_INVENTORY_ITEM_CREATED);
+
+  private int loadLimit = Integer.parseInt(System.getProperty("inventory.kafka.DataImportConsumer.loadLimit", "5"));
```
[the remark this answers is not in the document]
Comment (RuslanLavrov) [reply src/main/java/org/folio/inventory/DataImportConsumerVerticle.java]: moved to the separated methods

File: src/main/java/org/folio/inventory/DataImportConsumerVerticle.java [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+  public void start(Promise<Void> startPromise) {
+    JsonObject config = vertx.getOrCreateContext().config();
+    KafkaConfig kafkaConfig = KafkaConfig.builder()
+      .envId(config.getString(KAFKA_ENV))
+      .kafkaHost(config.getString(KAFKA_HOST))
+      .kafkaPort(config.getString(KAFKA_PORT))
+      .okapiUrl(config.getString(OKAPI_URL))
+      .replicationFactor(Integer.parseInt(config.getString(KAFKA_REPLICATION_FACTOR)))
+      .build();
+    LOGGER.debug("kafkaConfig: " + kafkaConfig);
```
[the remark this answers is not in the document]
Comment (RuslanLavrov) [reply src/main/java/org/folio/inventory/DataImportConsumerVerticle.java]: done

**Review by OleksiiKuzminov — APPROVED** [review]

**Pull request merged**